pkg/proxy/util/utils_linux.go

```diff
@@ -0,0 +1,73 @@
+// +build linux
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"net"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+)
+
+type NetlinkHandle interface {
+	AddrAdd(link netlink.Link, addr *netlink.Addr) error
```

Comment:
I would take `EnsureAddressBind()` and `UnbindAddress()` as interface functions, and hide AddrAdd, AddrDel, LinkByName.

Reply:
I'am trying to do so, But after I changed to this, the test cases are hard to write, because we can't mock netlink function in `EnsureAddressBind`.

Reply:
I can't understand. The parameter of `EnsureAddressBind()` has nothing to do with netlink. We can simply return nil in FakeInterface.EnsureAddressBind()

Reply:
I mean I can't write a test case to test `EnsureAddressBind` itself.

Reply:
Don't worry much about it.